carbon-design-system/carbon-for-ibm-dotcom
# fix(card-in-card): prevent new tab from opening twice on Firefox

### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7941

### Description

- This PR modifies the `onclick` function to query select the `dds-card-in-card-image` instead of the whole card.

### Changelog

**New**

- Adjust the WC sandbox example to reflect the up to date markup.

**Test**

- Click on the WC sandbox link and navigate to the `web-components/examples/codesandbox/components/card-in-card`.
- Open this sandbox preview in FF
- Click on the CTA card located on the bottom right-hand corner.
- Ensure that the new tab is opening just once.
